client/src/App.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
@rgagnon2 Nice and clean code! 🎉 🎉 
Readable and is easy to understand what the code is trying to do.